samples/net/cloud/aws_iot_mqtt/src/main.c

```diff
@@ -0,0 +1,494 @@
+/*
+ * Copyright (c) 2023 Lucas Dietrich <ld.adecy@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "creds/creds.h"
+#include "dhcp.h"
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <poll.h>
+#include <unistd.h>
+
+#include <zephyr/data/json.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/net/dns_resolve.h>
+#include <zephyr/net/mqtt.h>
+#include <zephyr/net/sntp.h>
+#include <zephyr/net/tls_credentials.h>
+#include <zephyr/random/rand32.h>
+
+
+#if defined(CONFIG_MBEDTLS_MEMORY_DEBUG)
+#include <mbedtls/memory_buffer_alloc.h>
+#endif
+
+#include <sys/socket.h>
+LOG_MODULE_REGISTER(aws, LOG_LEVEL_DBG);
+
+#define SNTP_SERVER "0.pool.ntp.org"
+
+#define AWS_BROKER_PORT "8883"
+
+#define MQTT_BUFFER_SIZE 256u
+#define APP_BUFFER_SIZE	 4096u
+
+#define MAX_RETRIES	    10u
+#define BACKOFF_EXP_BASE_MS 1000u
+#define BACKOFF_EXP_MAX_MS  60000u
+#define BACKOFF_CONST_MS    5000u
+
+static struct sockaddr_in aws_broker;
+
+static uint8_t rx_buffer[MQTT_BUFFER_SIZE];
+static uint8_t tx_buffer[MQTT_BUFFER_SIZE];
+static uint8_t buffer[APP_BUFFER_SIZE]; /* Shared between published and received messages */
+
+static struct mqtt_client client_ctx;
+
+static const char mqtt_client_name[] = CONFIG_AWS_THING_NAME;
+
+static uint32_t messages_received_counter;
+static bool do_publish;	  /* Trigger client to publish */
+static bool do_subscribe; /* Trigger client to subscribe */
+
+#define TLS_TAG_DEVICE_CERTIFICATE 1
+#define TLS_TAG_DEVICE_PRIVATE_KEY 1
+#define TLS_TAG_AWS_CA_CERTIFICATE 1
+
+static sec_tag_t sec_tls_tags[] = {
+	TLS_TAG_DEVICE_CERTIFICATE,
+};
+
+static int setup_credentials(void)
+{
+	int ret;
+
+	ret = tls_credential_add(TLS_TAG_DEVICE_CERTIFICATE, TLS_CREDENTIAL_SERVER_CERTIFICATE,
+				 public_cert, public_cert_len);
+	if (ret < 0) {
+		LOG_ERR("Failed to add device certificate: %d", ret);
+		goto exit;
+	}
+
+	ret = tls_credential_add(TLS_TAG_DEVICE_PRIVATE_KEY, TLS_CREDENTIAL_PRIVATE_KEY,
+				 private_key, private_key_len);
+	if (ret < 0) {
+		LOG_ERR("Failed to add device private key: %d", ret);
+		goto exit;
+	}
+
+	ret = tls_credential_add(TLS_TAG_AWS_CA_CERTIFICATE, TLS_CREDENTIAL_CA_CERTIFICATE, ca_cert,
+				 ca_cert_len);
+	if (ret < 0) {
+		LOG_ERR("Failed to add device private key: %d", ret);
+		goto exit;
+	}
+
+exit:
+	return ret;
+}
+
+static int subscribe_topic(void)
+{
+	int ret;
+
```

Comment:
Those empty lines between variables defined are not needed.

Reply:
Fixed, do you identify these style inconsistencies on your own or do you have a script for it?

Reply:
I'm afraid this is not verified by the scripts we have, so yes, manual spot.